test/functional/teacher_sets_controller_test.rb

```diff
@@ -13,39 +14,15 @@ class TeacherSetsControllerTest < ActionController::TestCase
     assert_not_nil assigns(:teacher_sets)
   end
 
-  test "should get new" do
-    get :new
+  test "should show teacher_set" do
+    get :show, params: { id: @teacher_set.id }
```

Comment:
What would be the difference between the "should get new" and the "should show teacher_set"?  Why did one of them go away?

Reply:
@daryanypl Inn teacherset_controller 'get' method is no there. So removed get method.